community/opendoas/files/doas.conf

```diff
@@ -0,0 +1,6 @@
+### Select one of the following. Deleting these may cause package installation
+### and removal to function unexpectedly
+
+permit setenv { KISS_FORCE } :wheel cmd kiss
+#permit persist setenv { KISS_FORCE } :wheel cmd kiss
+#permit nopass setenv { KISS_FORCE } :wheel cmd kiss
```

Comment:
Explanation of these three lines?

Also note: The package manager supports other environment variables such as `KISS_PATH`, `KISS_PID`, etc.

Reply:
permits kiss to be run as root if the user belongs to `wheel`, maintaining KISS_FORCE from the invoking user's environment. `persist` caches the privilege escalation for some indeterminate amount of time, and `nopass` allows it to be run without a password. I just commented these out in the same vein as e.g. /etc/sudoers.

Regarding environmental variables, `KISS_FORCE` seems to be the only one used when privileges are escalated (under installation and removal), hence KISS_FORCE="$KISS_FORCE" on lines 969 and 971, with the other environmental variables used only for unprivileged actions such building, etc., but please correct me if I'm wrong.

Reply:
:+1:

Reply:
I've just learned that `persist` requires an additional `configure` option due to it being an unstable feature on Linux, so I've removed that commented-out line